src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +166,19 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
-
+    
+    private static String getPersonPrintableString(ReadOnlyPerson person) {
+    	return getPrintableString(person.getName(), person.getPhone(), person.getEmail(), person.getAddress());
+	}
+
+    private static String getPrintableString(Printable... printables){
+    	String line = "";
+    	for(Printable printItem: printables){
+    		line += printItem.getPrintableString();
```

Comment:
concatenating a string within a loop in this manner may not be the most efficient way of joining Strings. You may want to look into StringBuilder or other APIs to do this inside a loop